error: reference to packed field is unaligned
  --> $DIR/unaligned_references.rs:18:5
   |
18 |     &a.f; //~ ERROR reference to packed field is unaligned
   |     ^^^^
   |
note: the lint level is defined here
  --> $DIR/unaligned_references.rs:1:11
   |
1  | #![forbid(unaligned_references)]
   |           ^^^^^^^^^^^^^^^^^^^^
   = note: fields of packed structs are not properly aligned, and creating a misaligned reference is undefined behavior (even if that reference is never dereferenced)

error: reference to packed field is unaligned
  --> $DIR/unaligned_references.rs:21:5
   |
21 |     &b.f; //~ ERROR reference to packed field is unaligned
   |     ^^^^
   |
   = note: fields of packed structs are not properly aligned, and creating a misaligned reference is undefined behavior (even if that reference is never dereferenced)
